app/Libraries/Payments/PaypalPaymentProcessor.php

```diff
@@ -70,16 +70,16 @@ public function getNotificationType(): string
         static $rejectedStatuses = ['Declined', 'Denied', 'Expired', 'Failed', 'Voided'];
 
         $status = $this->getNotificationTypeRaw();
-        if (in_array($status, $paymentStatuses, true)) {
+        if ($this->shouldIgnore($status)) {
+            return NotificationType::IGNORED;
+        } elseif (in_array($status, $paymentStatuses, true)) {
```

Comment:
semi unrelated but this looks like a job for switch/match.

Reply:
I don't think it's worth switching it right now